### ISSUE TITLE
Incorrect keyframes syntax breaks Safari9's CSS parser

### ISSUE THREAD

**Issue by Reinmar** [madebysource/lesshat#164]
There's a problem in Safari 9 now, that CSS generated by LESS+LESSHat breaks the entire parser if the `keyframes()` function was used.

This is the rule that causes this issue:

```
.content h1:target lesshat-selector {
  -lh-property: 0; } 
@-webkit-keyframes targetLinkOpacity{ 0%{ opacity: 0 } 100%{ opacity: 1 }}
@-moz-keyframes targetLinkOpacity{ 0%{ opacity: 0 } 100%{ opacity: 1 }}
@-o-keyframes targetLinkOpacity{ 0%{ opacity: 0 } 100%{ opacity: 1 }}
@keyframes targetLinkOpacity{ 0%{ opacity: 0 } 100%{ opacity: 1 };
}
```

It's generated from: 

```
...
&:target {
	.keyframes( ~'targetLinkOpacity, 0%{ opacity: 0 } 100%{ opacity: 1 }' );
}
```

The problem is in the semicolon at the end of the last keyframe. After manually removing it everything is working fine.

This is a critical issue because Safari totally gives up on parsing once it encounters a syntax issue. I guess that this issue is related to #93. That semicolon seems to terminate the `-lh-property: 0...` value that LESS Hat uses to workaround the inability to generate keyframes in LESS.

**Comment by hashplus** [comment]
+1

**Comment by maeertin** [comment]
+1

**Comment by emilsall** [comment]
+1

**Comment by vincentboiardt** [comment]
+1

**Comment by Npahlfer** [comment]
+1

**Comment by lpcustom** [comment]
+1

**Comment by fr-olivier** [comment]
+1

**Comment by laicosly** [comment]
+1

**Comment by shleyze** [comment]
+1

**Comment by fredck** [comment]
+1

**Comment by oleq** [comment]
:+1:

**Comment by mlewand** [comment]
+1

**Comment by danjford** [comment]
+1

**Comment by globexdesigns** [comment]
+1

**Comment by munkacsimark** [comment]
+1

**Comment by uberTof** [comment]
Yup, our entire web app of 14k lines of CSS crumbles in Safari 9 because of one semi-colon due to lesshat keyframes mixin.

**Comment by jakubweber** [comment]
Big ONE +1! I spend hours to find this issue

**Comment by aukgit** [comment]
+1

**Comment by petrbrzek** [comment]
Sorry guys for waiting so long. I've merged #166 so it should be fixed. If the problem persist let me know and I'll open the issue again.

**Comment by mboudreau** [comment]
This has been fixed in 4.0.1.
